### PR TITLE
Move all on-boarding lessons into an onboarding folder

### LINKED ISSUE #1231: Move all on-boarding lessons into an onboarding folder
For the in-person participants, there is a [list on the main README](https://github.com/Techtonica/curriculum#week-1---onboarding--developer-tools) of intro topics.

## "Soft Skills"
These should all be moved  to `/onboarding`

- [x] Welcome, Tips, and Rules 
- [x] Expectations While at Techtonica
- [x] How to Learn
- [x] Professionalism
~- [ ] Roles in Tech~
- [x] How to Pair Program
- [x] Ergonomics 
- [x] Growth Mindset
- [x] Asking Good Questions

## Tooling
These should be moved to `/dev-tools`

- [x] Intro to Command Line Interface 
- [x] Advanced Command Line
- [x] Bash Profile
- [x] Vim
- [x] Operating Systems
- [x] Keyboard Shortcuts
- [x] Local Development with VS Code

Update all internal links that exist to point to the new locations.

### PR DESCRIPTION
**Pull request by CipherGirl**
Closes #1231 - Move all on-boarding lessons into an onboarding folder

Moved "Soft Skills" and "Tooling" items to `/onboarding` and `/dev-tools` respectively. Updated all internal links that exist to point to the new locations.

### CODE REVIEW

**Comment by CipherGirl** [comment]
@Kundan28 Thank you so much 😍
This is my first time contributing to open source and I'm really enjoying.

**Comment by Kundan28** [comment]
That's great to hear that this is your first time. 
Please resolve the conflicts in the `README.md` file

**Comment by CipherGirl** [comment]
I have resolved the conflicts. Please review.

**Comment by CipherGirl** [comment]
I'm sorry I messed up something here.. Can someone guide me through it?

**Comment by Kundan28** [comment]
@CipherGirl Nice work. Now the PR is ready to be merged.
@alodahl

**Comment by Kundan28** [comment]
@CipherGirl Please resolve the conflicts.

**Comment by CipherGirl** [comment]
@Kundan28 I'm a beginner. Can you guide how can I resolve this? These conflicts show that it can't be resolved in the browser.

**Comment by CipherGirl** [comment]
I guess I've resolved it. Hoping I've not messed up :|

File: career/roles-in-tech/roles-in-tech.md [hunk 1]
```diff
@@ -7,7 +7,11 @@ About 1 hour
 - 30 minutes for the Independent Practice
 
 ### Prerequisites
+<<<<<<< HEAD:onboarding/roles-in-tech.md
```
Comment (alodahl) [inline career/roles-in-tech/roles-in-tech.md]: could you please remove these merge conflict lines?

Comment (CipherGirl) [reply career/roles-in-tech/roles-in-tech.md]: @alodahl miss, the changes here is done according to issue #1205. Where the file professionalism.md is beign moved to /career/professionalism.md and in my issue I had to move this file to /onboadring/professionalism.md. Which one is correct?

File: onboarding/roles-in-tech.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Roles in Tech
```
[the remark this answers is not in the document]
Comment (CipherGirl) [reply onboarding/roles-in-tech.md]: This file is being moved according to issue #1205. Should I change to that?

File: career/roles-in-tech/roles-in-tech.md (hunk 1 above)
Comment (alodahl) [reply career/roles-in-tech/roles-in-tech.md]: Onboarding

File: onboarding/roles-in-tech.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alodahl) [reply onboarding/roles-in-tech.md]: Yes please change it. I spoke to Bill about it this week.

**Comment by alodahl** [comment]
Hi @CipherGirl , will you be able to address these last comments and resolve conflicts so we can merge? We haven’t heard from you in 9 days.

**Comment by CipherGirl** [comment]
@alodahl I'm sorry miss, I was sick for couple of days and got distracted from the workflow. I've resolved the conflicts. Please check.

File: onboarding/roles-in-tech.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alodahl) [reply onboarding/roles-in-tech.md]: Hi @CipherGirl , please change this one back and it's links as discussed.

**Comment by CipherGirl** [comment]
@alodahl miss, I'm sorry I missed that before, I've updated that. Please review.

**Comment by alodahl** [comment]
Great work!

**Comment by Kundan28** [comment]
Updated in scoresheet too.

**Pull request merged**